### PR TITLE
Release 1.2.3 + misc fixes

### PR DESCRIPTION
**Pull request by katsikora**

### CODE REVIEW

File: snakePipes/shared/defaults.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # permitted here.
 ################################################################################
 snakemakeOptions: ' --use-conda --conda-prefix /package/anaconda3/envs/ '
-tempDir: /data/extended/
+tempdir: /data/extended/
```
Comment (dpryan79) [inline snakePipes/shared/defaults.yaml]: Is there a reason you're changing this?

File: snakePipes/workflows/RNA-seq/Snakefile [hunk 2]
```diff
@@ -276,7 +276,8 @@ onstart:
 ################################################################################
 
 if not fromBAM:
-    localrules: FASTQ1, FASTQ2
+    localrules: FASTQ1, FASTQ2, Salmon_wasabi
```
Comment (dpryan79) [inline snakePipes/workflows/RNA-seq/Snakefile]: That might fix things randomly for us, but it's unlikely to do so for others.

File: snakePipes/shared/defaults.yaml (hunk 1 above)
Comment (katsikora) [reply snakePipes/shared/defaults.yaml]: Looks like a mistake in resolving merge conflicts. I'll fix that, thanks!

File: snakePipes/workflows/RNA-seq/Snakefile (hunk 2 above)
Comment (katsikora) [reply snakePipes/workflows/RNA-seq/Snakefile]: I'm aware of this, the issue is still open.

**Pull request merged**